app/addons/auth/components.react.jsx

```diff
@@ -154,12 +153,12 @@ define([
             <input id="password" type="password" ref="password" name="password" placeholder="Password"
               size="24" onChange={this.onChangePassword} value={this.state.password} />
             <br />
-            <input id="password-confirm" type="password" name="password_confirm" placeholder= "Verify Password"
+            <input id="password-confirm" type="password" name="password_confirm" placeholder= "Password confirmation"
```

Comment:
`placeholder= "Password confirmation"`

one whitespace that does not belong there